qt/widgets/instrumentview/src/InstrumentActor.cpp

```diff
@@ -60,6 +60,8 @@ bool isPhysicalView() {
 const size_t InstrumentActor::INVALID_INDEX =
     std::numeric_limits<size_t>::max();
 double InstrumentActor::m_tolerance = 0.00001;
+const double InstrumentActor::INVALID_VALUE =
+    -std::numeric_limits<double>::max();
```

Comment:
Could this be defined as a `constexpr` in the header file? 

`static constexpr double INVALID_VALUE = -std::numeric_limits<double>::max();`

Reply:
Yes, I've also moved a couple of other consts in that file as well.